src/Categories/Multi/Category/Cartesian/Representable.agda

```diff
@@ -0,0 +1,174 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Multi.Category.Cartesian.Representable where
+
+open import Data.List hiding ([_])
```

Comment:
I prefer to use `using` as much as possible. It actually makes a non-trivial difference in the size of .agdai files, and from there to other things. But if basically everything is used, it's ok to leave it off.